lib/ccf/structural_preprocessing/one_subject_prereq_checker.py

```diff
@@ -6,6 +6,10 @@
 
```

Comment:
I think all the changes here are part of the submitting jobs from "local" machine feature.